src/overloads/arrays.jl

```diff
@@ -0,0 +1,208 @@
+    t = LinearAlgebra.logdet(T)
+    return D(p, t)
+end
+function LinearAlgebra.logabsdet(A::AbstractMatrix{D}) where {D<:Dual}
+    P, T = split_dual_array(A)
+    p1, p2 = LinearAlgebra.logabsdet(P)
+    t1, t2 = LinearAlgebra.logabsdet(T)
+    return (D(p1, t1), D(p2, t2))
+end
+
+## Norm
+function LinearAlgebra.norm(A::AbstractArray{T}, p::Real=2) where {T<:AbstractTracer}
+    return second_order_or(A)
+end
+function LinearAlgebra.opnorm(A::AbstractArray{T}, p::Real=2) where {T<:AbstractTracer}
+    return first_order_or(A)
+end
+function LinearAlgebra.opnorm(A::AbstractMatrix{T}, p::Real=2) where {T<:AbstractTracer}
+    return first_order_or(A)
+end
+
+## Eigenvalues
+
+function LinearAlgebra.eigmax(
+    A::Union{T,AbstractMatrix{T}}; permute::Bool=true, scale::Bool=true
+) where {T<:AbstractTracer}
+    return second_order_or(A)
+end
+function LinearAlgebra.eigmin(
+    A::Union{T,AbstractMatrix{T}}; permute::Bool=true, scale::Bool=true
+) where {T<:AbstractTracer}
+    return second_order_or(A)
+end
+function LinearAlgebra.eigen(
+    A::AbstractMatrix{T};
+    permute::Bool=true,
+    scale::Bool=true,
+    sortby::Union{Function,Nothing}=nothing,
+) where {T<:AbstractTracer}
+    LinearAlgebra.checksquare(A)
+    n = size(A, 1)
+    t = second_order_or(A)
+    values = Fill(t, n)
+    vectors = Fill(t, n, n)
+    return LinearAlgebra.Eigen(values, vectors)
+end
+
+## Inverse
+function LinearAlgebra.inv(A::StridedMatrix{T}) where {T<:AbstractTracer}
+    LinearAlgebra.checksquare(A)
+    t = second_order_or(A)
+    return Fill(t, size(A)...)
+end
+function LinearAlgebra.pinv(
+    A::AbstractMatrix{T}; atol::Real=0.0, rtol::Real=0.0
+) where {T<:AbstractTracer}
+    n, m = size(A)
+    t = second_order_or(A)
+    return Fill(t, m, n)
+end
+
+## Division
+function LinearAlgebra.:\(
+    A::AbstractMatrix{T}, B::AbstractVecOrMat
+) where {T<:AbstractTracer}
+    Ainv = LinearAlgebra.pinv(A)
+    return Ainv * B
+end
+
+## Exponent
+function LinearAlgebra.exp(A::AbstractMatrix{T}) where {T<:AbstractTracer}
+    LinearAlgebra.checksquare(A)
+    n = size(A, 1)
+    t = second_order_or(A)
+    return Fill(t, n, n)
+end
+
+## Matrix power
+function LinearAlgebra.:^(A::AbstractMatrix{T}, p::Integer) where {T<:AbstractTracer}
+    LinearAlgebra.checksquare(A)
+    n = size(A, 1)
+    t = second_order_or(A)
+    return Fill(t, n, n)
+end
+
+#==============#
+# SparseArrays #
+#==============#
+
+# Conversion of matrices of tracers to SparseMatrixCSC has to be rewritten 
+# due to use of `count(_isnotzero, M)` in SparseArrays.jl
+#
+# Code modified from MIT licensed SparseArrays.jl source:
+# https://github.com/JuliaSparse/SparseArrays.jl/blob/45dfe459ede2fa1419e7068d4bda92d9d22bd44d/src/sparsematrix.jl#L901-L920
+# Copyright (c) 2009-2024: Jeff Bezanson, Stefan Karpinski, Viral B. Shah, and other contributors: https://github.com/JuliaLang/julia/contributors
+function SparseArrays.SparseMatrixCSC{Tv,Ti}(
+    M::StridedMatrix{Tv}
```

Comment:
why `Strided`?

Reply:
That's what LinearAlgebra.jl uses, so we need to subtype `StridedMatrix{Tv}`, `Tv<:AbstractTracer`.